File: Msys_Programs/MSys_Projects/LeetCode/Python/Two_Sum.py
class Solution:
    def twoSum(self,List, target):
        check = set()
        for i in range(len(List)):
            if List[i] in check:
                return [List.index(target - List[i]), i]
            else:
                check.add(target - List[i])


TS = Solution()
B = TS.twoSum([2, 7, 11, 15], 9)
print(B)

File: Msys_Programs/MSys_Projects/LeetCode/GoLang/Two_Sum.go
package main

import "fmt"

func twoSum(nums []int, target int) []int {
	for i, right := range nums {
		for j, left := range nums {
			if right+left == target && i != j {
				return []int{i, j}
			}
		}
	}
	return nil
}

func main() {
	B := []int{2, 7, 11, 15}
	Target := 9
	T := twoSum(B, Target)
	fmt.Println(T)
}
